provision/vvv-init.sh

```diff
@@ -153,4 +154,15 @@ if [ ! -z "${WP_PLUGINS}" ]; then
     done
 fi
 
+if [[ ! -z "${FORCE_CHANGE_WP_VERSION}" ]]; then
+  if [[ -f "${VVV_PATH_TO_SITE}/public_html/wp-includes/version.php" ]]; then
+    CURRENT_VERSION=`grep wp_version "${VVV_PATH_TO_SITE}/public_html/wp-includes/version.php" | awk -F "'" '{print $2}' | grep -E "\d+"`
```

Comment:
Running this command I get empty lines and not only the value, replacing the grep command with `awk "NF > 0"` for me was working.

Reply:
Thanks, @Mte90! I'll get this updated.